CombinatorialGames/Game/Birthday.lean

```diff
@@ -5,6 +5,7 @@ Authors: Violeta Hernández Palacios
 -/
 import Mathlib.Algebra.Order.Group.OrderIso
 import CombinatorialGames.Game.Ordinal
+import CombinatorialGames.Game.Specific
```

Comment:
Should theorems about birthdays/impartialness be moved to Specific?

Reply:
I think this file should preferably as low as possible in the import chain. We should be able to import special games in `Impartial` / `Birthday` / all those other files and prove whatever the relevant properties are, not the other way around, unless we want these simple games to be bloated behind a bloated file.